<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ULTRACOLD UB: Wave Packet Dispersion Laboratory</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="mplwindow" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>621</width>
      <height>461</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <layout class="QVBoxLayout" name="mplvl">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
    </layout>
   </widget>
   <widget class="QListWidget" name="mplfigs">
    <property name="geometry">
     <rect>
      <x>633</x>
      <y>287</y>
      <width>200</width>
      <height>181</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>16777215</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="interact" native="true">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>211</width>
      <height>271</height>
     </rect>
    </property>
    <widget class="QSlider" name="horizontalSlider">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>211</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-6</number>
     </property>
     <property name="maximum">
      <number>6</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>200</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>5</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Initial position WavePack:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>180</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Number of oscillations:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="start">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>240</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>START</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Yes</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>No</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Harmonic:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Excitation of the state:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>140</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>2</number>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>520</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider_simulation">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>581</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Simulation Time:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>500</y>
      <width>46</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonOn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>480</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Go on</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonBack">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>480</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Go back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonPause">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>480</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QWidget" name="interact_game" native="true">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>470</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <widget class="QRadioButton" name="radioButton_oscil">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Oscillation</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_game">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Guess Function</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="game" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>621</width>
      <height>171</height>
     </rect>
    </property>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>601</width>
       <height>121</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;The dark solitons under an harmonic trap behave as a normal particle would do in an harmonic potential. We know that in such case the particle follows a sinusoidal function for its position with time, so to say:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;	x(t)=A cos(wt+B)&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;with A being its amplitude, w its angular frequency and B an arbitrary phase.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Can you adjust the movement plotted from data to a function by introducing the coefficients A and w? Suppose B is 0.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_try">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>140</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Try</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_return">
     <property name="geometry">
      <rect>
       <x>524</x>
       <y>140</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Return to Plots</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_osci" native="true">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>140</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <widget class="QDoubleSpinBox" name="spin_amplitude">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>0</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <double>-6.000000000000000</double>
      </property>
      <property name="maximum">
       <double>6.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="spin_frequency">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>0</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <double>10.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_A">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>A:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_w">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>0</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>w:</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="mplwindow_2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>621</width>
      <height>421</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <layout class="QVBoxLayout" name="mplvl_2">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
    </layout>
   </widget>
   <widget class="QWidget" name="mplwindow_3" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>301</width>
      <height>271</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <layout class="QVBoxLayout" name="mplvl_3">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
    </layout>
   </widget>
   <widget class="QPushButton" name="ButtonDiffu">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>480</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Diffusion</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textDiffu">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>611</width>
      <height>161</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Diffusion is the process by which a certain magnitude is transported following a gradient. A gradient indicates the change in a variable (for example, temperature) with the change in another quantity (for example, distance). It is important to see that diffusion does not imply a mass transport like in advection or convection. &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Here we have an example of the thermal diffusion in 2D. We have a sheet that has a fixed temperature in its extremes and an initial thermal profile (gaussian). We can observe that exists a transport of temperature in the sheet, from high to low temperatures.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;At the left we can see the temperature profile in the sheet and clearly we can observe that exists a temperature gradient. At the right we have the cross-section of our temperature profile. &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonBackDiffu">
    <property name="geometry">
     <rect>
      <x>544</x>
      <y>590</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Return to plots</string>
    </property>
   </widget>
   <widget class="QWidget" name="mplwindow_4" native="true">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>90</y>
      <width>301</width>
      <height>271</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <layout class="QVBoxLayout" name="mplvl_5">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>875</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>651</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>760</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>start</sender>
   <signal>clicked()</signal>
   <receiver>mplfigs</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>717</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>699</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>start</sender>
   <signal>clicked()</signal>
   <receiver>mplwindow</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>735</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_simulation</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_4</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>548</y>
    </hint>
    <hint type="destinationlabel">
     <x>108</x>
     <y>529</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>start</sender>
   <signal>clicked()</signal>
   <receiver>mplfigs</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>764</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>764</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
